daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/iterable/ExprIterable.scala

```diff
@@ -12,7 +12,7 @@ private[validation] object ExprIterable {
 
   private[iterable] def iterator(x: Expr): Iterator[Expr] = {
     x match {
-      case EVar(_) | EBuiltin(_) | EPrimCon(_) | EPrimLit(_) | EVal(_) | EEnumCon(_, _) |
+      case EVar(_) | EBuiltin(_) | EBuiltinCon(_) | EBuiltinLit(_) | EVal(_) | EEnumCon(_, _) |
```

Comment:
I think we should also rename `EBuiltin`=>`EBuiltinFun` or `EBuiltinFunction`

Reply:
make sens.

Reply:
would you mind I make that in a follow up PR ?